fineract-core/src/main/java/org/apache/fineract/infrastructure/core/serialization/JsonParserHelper.java

```diff
@@ -459,28 +462,31 @@ public LocalTime extractLocalTimeNamed(final String parameterName, final JsonEle
         return value;
     }
 
-    public LocalDateTime extractLocalDateTimeNamed(final String parameterName, final JsonElement element, final String timeFormat,
+    public LocalDateTime extractLocalDateTimeNamed(final String parameterName, final JsonElement element, String timeFormat,
             final Locale clientApplicationLocale, final Set<String> parametersPassedInCommand) {
         LocalDateTime value = null;
-        String timeValueAsString = null;
+        String timeValueAsString;
         if (element.isJsonObject()) {
             final JsonObject object = element.getAsJsonObject();
             if (object.has(parameterName) && object.get(parameterName).isJsonPrimitive()) {
                 parametersPassedInCommand.add(parameterName);
 
                 try {
-                    DateTimeFormatter timeFormtter = DateTimeFormatter.ofPattern(timeFormat);
+                    timeFormat = timeFormat.replace("y", "u");
```

Comment:
I don't think we need this anymore, do we?

Reply:
it's still necessary since I haven't changed the type everywhere. The rest will be coming in a furhter PR.